src/UI/DamageChartController.java

```diff
@@ -0,0 +1,84 @@
+package UI;
+
+
+
+import java.awt.Color;
+
+import CommandingOfficers.Commander;
+import Engine.IController;
+import Engine.OptionSelector;
+import UI.InputHandler.InputAction;
+import UI.UIUtils.Faction;
+import Units.UnitModel;
+
+public class DamageChartController implements IController
+{
+  OptionSelector targetSelector;
+  OptionSelector shooterSelector;
+  public final Faction shooters, targets;
+  public final Color shooterColor, targetColor;
+  public final UnitModel[] units;
+  public boolean outOfAmmo = false;
+
+  public DamageChartController(Commander attacker, Commander defender)
+  {
+    // TODO: Account for variable unit model count?
+    units = attacker.unitModels.toArray(new UnitModel[0]);
+    shooters = attacker.faction;
+    targets = defender.faction;
+    shooterColor = attacker.myColor;
+    targetColor = defender.myColor;
+    targetSelector = new OptionSelector(units.length);
+    shooterSelector = new OptionSelector(units.length);
+  }
+  public DamageChartController(UnitModel[] models, Faction attackers, Faction defenders, Color attackerColor, Color defenderColor)
+  {
+    units = models;
+    shooters = attackers;
+    targets = defenders;
+    shooterColor = attackerColor;
+    targetColor = defenderColor;
+    targetSelector = new OptionSelector(units.length);
+    shooterSelector = new OptionSelector(units.length);
+  }
+
+  @Override
+  public boolean handleInput(InputAction action)
+  {
+    boolean done = false;
+    switch(action)
+    {
+      case SEEK:
+        outOfAmmo = !outOfAmmo;
```

Comment:
Would be good to have a visual indicator that this option is available.

Reply:
There's no real estate for it, at present, unless you expand the screen.
Even something non-obvious like "press Q" wouldn't really fit even in the top corner.

I guess it'd be possible to just sacrifice the real estate for the prompt *sometimes*, and have a hidden config setting (that gets automatically toggled when you use Q here) to hide the prompt and use all the space?